src/ast/irbuilderbpf.cpp

```diff
@@ -2212,10 +2212,10 @@ void IRBuilderBPF::CreatePerCpuMapElemInit(Map &map,
   CreateLifetimeEnd(initValue);
 }
 
-void IRBuilderBPF::CreatePerCpuMapElemAdd(Map &map,
-                                          Value *key,
-                                          Value *val,
-                                          const Location &loc)
+Value *IRBuilderBPF::CreatePerCpuMapElemAdd(Map &map,
```

Comment:
These changes aren't necessary. Without the anonymous map, we can simply embed the update logic for each of the aggregation functions (`min`/`max`, `sum`, `avg`) in the codegen for `tseries`. This avoids this patch and reduces overall changes.